### PR TITLE
Tiny improvements.

### PR DESCRIPTION
**Pull request by irfanevrens**
- Formatting Fixes to API objects
- Fixing documentation
- Fixed Missing Properties

### CODE REVIEW

File: lib/PayPal/Core/PayPalHttpConnection.php [hunk 1]
```diff
@@ -93,6 +93,7 @@ public function execute($data)
         switch ($this->httpConfig->getMethod()) {
             case 'POST':
                 curl_setopt($ch, CURLOPT_POST, true);
+                break;
```
Comment (jaypatel512) [inline lib/PayPal/Core/PayPalHttpConnection.php]: This is not true. The break is missing for a reason, as the all the above cases should be setting below option.

**Comment by jaypatel512** [comment]
Thanks for the pull request. I will definitely have a look and merge if everything looks good.

File: lib/PayPal/Core/PayPalHttpConnection.php (hunk 1 above)
Comment (irfanevrens) [inline lib/PayPal/Core/PayPalHttpConnection.php]: If so, there is a logical problem I think. 
if method is POST then
```
curl_setopt($ch, CURLOPT_POST, true);
curl_setopt($ch, CURLOPT_POSTFIELDS, $data);
```
and if method is one of [POST, PUT, PATCH, DELETE] then

```
curl_setopt($ch, CURLOPT_POSTFIELDS, $data);
```

am I right?

Comment (jaypatel512) [inline lib/PayPal/Core/PayPalHttpConnection.php]: No, this is how it works.

If POST: 
```
curl_setopt($ch, CURLOPT_POST, true);
curl_setopt($ch, CURLOPT_POSTFIELDS, $data);
```

IF PUT, PATCH, DELETE
```
curl_setopt($ch, CURLOPT_POSTFIELDS, $data);
```

Comment (irfanevrens) [inline lib/PayPal/Core/PayPalHttpConnection.php]: I repeat this code for a missing break in not empty case;
```
curl_setopt($ch, CURLOPT_POSTFIELDS, $data);
```

**Comment by jaypatel512** [comment]
This looks good now ! :+1: I will merge it soon. 

Thanks for your Pull Request.

**Comment by jaypatel512** [comment]
I was able to successfully merge your request. That also prompted me to fix few more similar issues in tests, and apis. 

Thanks for making our SDK better :)

**Comment by irfanevrens** [comment]
You are welcome @jaypatel512

**Pull request merged**